stacks/api/bucket/history/history.controller.spec.ts

```diff
@@ -211,4 +220,20 @@ describe("History Acceptance", () => {
       description: "current description"
     });
   });
+  it("should clear histories for given bucketId", async () => {
+    const response = await req.delete(`/bucket/${bucketId}/history`);
+    expect(response.statusCode).toEqual(204);
+    expect(response.body).toEqual(undefined);
+
+    const {body: deletedHistories} = await req.get(`/bucket/${bucketId}/history/${documentId}`, {});
+    expect(deletedHistories).toEqual([]);
```

Comment:
The body should be undefined.

Reply:
Shouldn't be an empty array if there isn't any history? Also, I didn't make any changes to this endpoint logic.